make/app.mk

```diff
@@ -155,7 +155,7 @@ upload: all
 
 # Check that the executable exists.
 exe-exists:
-	if [ ! -e $(EXE) ] ; then \
+	@if [ ! -e $(EXE) ] ; then \
```

Comment:
The build system expects the user to give `-s` to make for silent output. I wish make was silent by default and had an option for verbose output, but as far as I know it does not.

Reply:
I know, in general the verbosity of your make files are ok, only this one just annoyed me :) I don't like to see such _if_ statements in console log because the conditional message is printed in both cases (for true condition it is printed even twice).

Reply:
I see, let's keep your change. Recipes just printing a message could as well be forced silent. It's very unlikely that the user want to copy these commands and run them manually in the terminal.